src/Form.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import Input from "./Input";
+import { createNewConference } from "./api";
+import { useHistory } from "react-router-dom";
+
+const Form = () => {
+  let history = useHistory();
+  const [conference, setConference] = useState({
+    name: "",
+    dateTime: "",
+    city: "",
+    description: "",
+    topic: ""
+  });
+
+  const handleChange = event => {
+    const value = event.target.value;
+    setState({ ...conference, [event.target.name]: value });
+  };
+
+  const submitForm = async () => {
+    const newConference = {
+      ...conference,
+      dateTime: conference.dateTime + ":00Z"
+    };
+    try {
+      await createNewConference(newConference);
+      history.push("/");
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  return (
+    <form
+      onSubmit={ev => {
+        ev.preventDefault();
+        submitForm();
+      }}
+    >
+      <Input
+        id="name"
+        label="Conference Name: "
+        type="text"
+        name="name"
+        value={conference.name}
+        onChange={handleChange}
+        required
+        maxLength="80"
+      />
+      <br />
+      <Input
+        label="Date and Time: "
+        type="datetime-local"
```

Comment:
Have you tried using this on safari or firefox? I'm not 100% sure it is supported may be worth checking https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local

Reply:
Just did it, not supported by safari, I don't have Firefox to check that though

Reply:
Harry has said we don't have to support any browsers other than Chrome so I guess this is okay

Reply:
Yeah this isn't a problem for this project, just wanted to highlight how different browsers are, its not a blocker